App crashes on taping on the image of products.

<img src="https://user-images.githubusercontent.com/25099208/36173767-1d499e66-1130-11e8-849d-75392bd61b93.png" height="560"/>

**Steps to reproduce:** 

1) Open details of any Product which has its image.
2) Tap on the image.

**Stack Trace:** 

02-14 02:24:49.471 27357-27357/openfoodfacts.github.scrachx.openfood E/AndroidRuntime: FATAL EXCEPTION: main
                                                                                       Process: openfoodfacts.github.scrachx.openfood, PID: 27357
                                                                                       java.lang.RuntimeException: Unable to start activity ComponentInfo{openfoodfacts.github.scrachx.openfood/openfoodfacts.github.scrachx.openfood.views.FullScreenImage}: android.view.InflateException: Binary XML file line #7: Binary XML file line #7: Error inflating class uk.co.senab.photoview.PhotoView
                                                                                           at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2455)
                                                                                           at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2515)
                                                                                           at android.app.ActivityThread.access$1000(ActivityThread.java:154)
                                                                                           at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1379)
                                                                                           at android.os.Handler.dispatchMessage(Handler.java:102)
                                                                                           at android.os.Looper.loop(Looper.java:157)
                                                                                           at android.app.ActivityThread.main(ActivityThread.java:5555)
                                                                                           at java.lang.reflect.Method.invoke(Native Method)
                                                                                           at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:745)
                                                                                           at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:635)
                                                                                        Caused by: android.view.InflateException: Binary XML file line #7: Binary XML file line #7: Error inflating class uk.co.senab.photoview.PhotoView
                                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:543)
                                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:427)
                                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:374)
                                                                                           at android.support.v7.app.AppCompatDelegateImplV9.setContentView(AppCompatDelegateImplV9.java:287)
                                                                                           at android.support.v7.app.AppCompatActivity.setContentView(AppCompatActivity.java:139)
                                                                                           at openfoodfacts.github.scrachx.openfood.views.BaseActivity.setContentView(**BaseActivity.java:31**)
                                                                                           at openfoodfacts.github.scrachx.openfood.views.FullScreenImage.onCreate(**FullScreenImage.java:27**)
                                                                                           at android.app.Activity.performCreate(Activity.java:6357)
                                                                                           at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1108)
                                                                                           at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2408)
                                                                                           at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2515) 
                                                                                           at android.app.ActivityThread.access$1000(ActivityThread.java:154) 
                                                                                           at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1379) 
                                                                                           at android.os.Handler.dispatchMessage(Handler.java:102) 
                                                                                           at android.os.Looper.loop(Looper.java:157) 
                                                                                           at android.app.ActivityThread.main(ActivityThread.java:5555) 
                                                                                           at java.lang.reflect.Method.invoke(Native Method) 
                                                                                           at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:745) 
                                                                                           at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:635) 
                                                                                        Caused by: android.view.InflateException: Binary XML file line #7: Error inflating class uk.co.senab.photoview.PhotoView
                                                                                           at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:780)
                                                                                           at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:708)
                                                                                           at android.view.LayoutInflater.rInflate(LayoutInflater.java:839)
                                                                                           at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:802)
                                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:519)
                                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:427) 
                                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:374) 
                                                                                           at android.support.v7.app.AppCompatDelegateImplV9.setContentView(AppCompatDelegateImplV9.java:287) 
                                                                                           at android.support.v7.app.AppCompatActivity.setContentView(AppCompatActivity.java:139) 
                                                                                           at openfoodfacts.github.scrachx.openfood.views.BaseActivity.setContentView(BaseActivity.java:31) 
                                                                                           at openfoodfacts.github.scrachx.openfood.views.FullScreenImage.onCreate(FullScreenImage.java:27) 
                                                                                           at android.app.Activity.performCreate(Activity.java:6357) 
                                                                                           at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1108) 
                                                                                           at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2408) 
                                                                                           at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2515) 
                                                                                           at android.app.ActivityThread.access$1000(ActivityThread.java:154) 
                                                                                           at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1379) 
                                                                                           at android.os.Handler.dispatchMessage(Handler.java:102) 
                                                                                           at android.os.Looper.loop(Looper.java:157) 
                                                                                           at android.app.ActivityThread.main(ActivityThread.java:5555) 
                                                                                           at java.lang.reflect.Method.invoke(Native Method) 
                                                                                           at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:745) 
                                                                                           at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:635) 
                                                                                        Caused by: java.lang.ClassNotFoundException: Didn't find class "uk.co.senab.photoview.PhotoView" on path: DexPathList[[zip file "/data/app/openfoodfacts.github.scrachx.openfood-1/base.apk"],nativeLibraryDirectories=[/data/app/openfoodfacts.github.scrachx.openfood-1/lib/arm64, /vendor/lib64, /system/lib64]]
                                                                                           at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:56)
                                                                                           at java.lang.ClassLoader.loadClass(ClassLoader.java:511)
                                                                                           at java.lang.ClassLoader.loadClass(ClassLoader.java:469)
                                                                                           at android.view.LayoutInflater.createView(LayoutInflater.java:587)
                                                                                           at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:768)
                                                                                           at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:708) 
                                                                                           at android.view.LayoutInflater.rInflate(LayoutInflater.java:839) 
                                                                                           at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:802) 
                                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:519) 
                                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:427) 
                                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:374) 
                                                                                           at android.support.v7.app.AppCompatDelegateImplV9.setContentView(AppCompatDelegateImplV9.java:287) 
                                                                                           at android.support.v7.app.AppCompatActivity.setContentView(AppCompatActivity.java:139) 
                                                                                           at openfoodfacts.github.scrachx.openfood.views.BaseActivity.setContentView(BaseActivity.java:31) 
                                                                                           at openfoodfacts.github.scrachx.openfood.views.FullScreenImage.onCreate(FullScreenImage.java:27) 
                                                                                           at android.app.Activity.performCreate(Activity.java:6357) 
                                                                                           at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1108) 
                                                                                           at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2408) 
                                                                                           at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2515) 
                                                                                           at android.app.ActivityThread.access$1000(ActivityThread.java:154) 
                                                                                           at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1379) 
                                                                                           at android.os.Handler.dispatchMessage(Handler.java:102) 
                                                                                           at android.os.Looper.loop(Looper.java:157) 
                                                                                           at android.app.ActivityThread.main(ActivityThread.java:5555) 
                                                                                           at java.lang.reflect.Method.invoke(Native Method) 
                                                                                           at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:745) 
                                                                                           at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:635) 
                                                                                       	Suppressed: java.lang.ClassNotFoundException: uk.co.senab.photoview.PhotoView
                                                                                           at java.lang.Class.classForName(Native Method)
                                                                                           at java.lang.BootClassLoader.findClass(ClassLoader.java:781)
                                                                                           at java.lang.BootClassLoader.loadClass(ClassLoader.java:841)
                                                                                           at java.lang.ClassLoader.loadClass(ClassLoader.java:504)
                                                                                           		... 25 more
                                                                                        Caused by: java.lang.NoClassDefFoundError: Class not found using the boot class loader; no stack trace available
02-14 02:24:49.471 27357-27357/openfoodfacts.github.scrachx.openfood E/AndroidRuntime: FATAL EXCEPTION: main
                                                                                       Process: openfoodfacts.github.scrachx.openfood, PID: 27357
                                                                                       java.lang.RuntimeException: Unable to start activity ComponentInfo{openfoodfacts.github.scrachx.openfood/openfoodfacts.github.scrachx.openfood.views.FullScreenImage}: android.view.InflateException: Binary XML file line #7: Binary XML file line #7: Error inflating class uk.co.senab.photoview.PhotoView
                                                                                           at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2455)
                                                                                           at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2515)
                                                                                           at android.app.ActivityThread.access$1000(ActivityThread.java:154)
                                                                                           at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1379)
                                                                                           at android.os.Handler.dispatchMessage(Handler.java:102)
                                                                                           at android.os.Looper.loop(Looper.java:157)
                                                                                           at android.app.ActivityThread.main(ActivityThread.java:5555)
                                                                                           at java.lang.reflect.Method.invoke(Native Method)
                                                                                           at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:745)
                                                                                           at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:635)
                                                                                        Caused by: android.view.InflateException: Binary XML file line #7: Binary XML file line #7: Error inflating class uk.co.senab.photoview.PhotoView
                                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:543)
                                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:427)
                                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:374)
                                                                                           at android.support.v7.app.AppCompatDelegateImplV9.setContentView(AppCompatDelegateImplV9.java:287)
                                                                                           at android.support.v7.app.AppCompatActivity.setContentView(AppCompatActivity.java:139)
                                                                                           at openfoodfacts.github.scrachx.openfood.views.BaseActivity.setContentView(BaseActivity.java:31)
                                                                                           at openfoodfacts.github.scrachx.openfood.views.FullScreenImage.onCreate(FullScreenImage.java:27)
                                                                                           at android.app.Activity.performCreate(Activity.java:6357)
                                                                                           at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1108)
                                                                                           at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2408)
                                                                                           at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2515) 
                                                                                           at android.app.ActivityThread.access$1000(ActivityThread.java:154) 
                                                                                           at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1379) 
                                                                                           at android.os.Handler.dispatchMessage(Handler.java:102) 
                                                                                           at android.os.Looper.loop(Looper.java:157) 
                                                                                           at android.app.ActivityThread.main(ActivityThread.java:5555) 
                                                                                           at java.lang.reflect.Method.invoke(Native Method) 
                                                                                           at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:745) 
                                                                                           at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:635) 
                                                                                        Caused by: android.view.InflateException: Binary XML file line #7: Error inflating class uk.co.senab.photoview.PhotoView
                                                                                           at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:780)
                                                                                           at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:708)
                                                                                           at android.view.LayoutInflater.rInflate(LayoutInflater.java:839)
                                                                                           at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:802)
                                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:519)
                                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:427) 
                                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:374) 
                                                                                           at android.support.v7.app.AppCompatDelegateImplV9.setContentView(AppCompatDelegateImplV9.java:287) 
                                                                                           at android.support.v7.app.AppCompatActivity.setContentView(AppCompatActivity.java:139) 
                                                                                           at openfoodfacts.github.scrachx.openfood.views.BaseActivity.setContentView(BaseActivity.java:31) 
                                                                                           at openfoodfacts.github.scrachx.openfood.views.FullScreenImage.onCreate(FullScreenImage.java:27) 
                                                                                           at android.app.Activity.performCreate(Activity.java:6357) 
                                                                                           at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1108) 
                                                                                           at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2408) 
                                                                                           at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2515) 
                                                                                           at android.app.ActivityThread.access$1000(ActivityThread.java:154) 
                                                                                           at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1379) 
                                                                                           at android.os.Handler.dispatchMessage(Handler.java:102) 
                                                                                           at android.os.Looper.loop(Looper.java:157) 
                                                                                           at android.app.ActivityThread.main(ActivityThread.java:5555) 
                                                                                           at java.lang.reflect.Method.invoke(Native Method) 
                                                                                           at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:745) 
                                                                                           at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:635) 
                                                                                        Caused by: java.lang.ClassNotFoundException: Didn't find class "uk.co.senab.photoview.PhotoView" on path: DexPathList[[zip file "/data/app/openfoodfacts.github.scrachx.openfood-1/base.apk"],nativeLibraryDirectories=[/data/app/openfoodfacts.github.scrachx.openfood-1/lib/arm64, /vendor/lib64, /system/lib64]]
                                                                                           at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:56)
                                                                                           at java.lang.ClassLoader.loadClass(ClassLoader.java:511)
                                                                                           at java.lang.ClassLoader.loadClass(ClassLoader.java:469)
                                                                                           at android.view.LayoutInflater.createView(LayoutInflater.java:587)
                                                                                           at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:768)
                                                                                           at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:708) 
                                                                                           at android.view.LayoutInflater.rInflate(LayoutInflater.java:839) 
                                                                                           at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:802) 
                                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:519) 
                                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:427) 
                                                                                           at android.view.LayoutInflater.inflate(LayoutInflater.java:374) 
                                                                                           at android.support.v7.app.AppCompatDelegateImplV9.setContentView(AppCompatDelegateImplV9.java:287) 
                                                                                           at android.support.v7.app.AppCompatActivity.setContentView(AppCompatActivity.java:139) 
                                                                                           at openfoodfacts.github.scrachx.openfood.views.BaseActivity.setContentView(**BaseActivity.java:31**) 
                                                                                           at openfoodfacts.github.scrachx.openfood.views.FullScreenImage.onCreate(**FullScreenImage.java:27**) 
                                                                                           at android.app.Activity.performCreate(Activity.java:6357) 
                                                                                           at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1108) 
                                                                                           at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2408) 
                                                                                           at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2515) 
                                                                                           at android.app.ActivityThread.access$1000(ActivityThread.java:154) 
                                                                                           at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1379) 
                                                                                           at android.os.Handler.dispatchMessage(Handler.java:102) 
                                                                                           at android.os.Looper.loop(Looper.java:157) 
                                                                                           at android.app.ActivityThread.main(ActivityThread.java:5555) 
                                                                                           at java.lang.reflect.Method.invoke(Native Method) 
                                                                                           at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:745) 
                                                                                           at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:635) 
                                                                                       	Suppressed: java.lang.ClassNotFoundException: uk.co.senab.photoview.PhotoView
                                                                                           at java.lang.Class.classForName(Native Method)
                                                                                           at java.lang.BootClassLoader.findClass(ClassLoader.java:781)
                                                                                           at java.lang.BootClassLoader.loadClass(ClassLoader.java:841)
                                                                                           at java.lang.ClassLoader.loadClass(ClassLoader.java:504)
                                                                                           		... 25 more
                                                                                        Caused by: java.lang.NoClassDefFoundError: Class not found using the boot class loader; no stack trace available


**I will work on it.**

--- a/app/src/main/java/openfoodfacts/github/scrachx/openfood/views/FullScreenImage.java
+++ b/app/src/main/java/openfoodfacts/github/scrachx/openfood/views/FullScreenImage.java
@@ -18,7 +18,7 @@ import static org.apache.commons.lang3.StringUtils.isNotEmpty;
 public class FullScreenImage extends BaseActivity {
 
     @BindView(R.id.imageViewFullScreen)
-    ImageView mImageView;
+    PhotoView mPhotoView;
     PhotoViewAttacher mAttacher;
 
     @Override
@@ -30,12 +30,12 @@ public class FullScreenImage extends BaseActivity {
         Intent intent = getIntent();
         String imageurl = intent.getExtras().getString("imageurl");
 
-        mAttacher = new PhotoViewAttacher(mImageView);
+        mAttacher = new PhotoViewAttacher(mPhotoView);
 
         if (isNotEmpty(imageurl)) {
             Picasso.with(this)
                     .load(imageurl)
-                    .into(mImageView, new Callback() {
+                    .into(mPhotoView, new Callback() {
                         @Override
                         public void onSuccess() {
                             mAttacher.update();

--- a/app/src/main/res/layout/activity_full_screen_image.xml
+++ b/app/src/main/res/layout/activity_full_screen_image.xml
@@ -5,7 +5,7 @@
     android:background="@android:color/black"
     android:orientation="vertical">
 
-    <ImageView
+    <com.github.chrisbanes.photoview.PhotoView
         android:id="@+id/imageViewFullScreen"
         android:layout_width="match_parent"
         android:layout_height="match_parent"

